src/tools/tests/goldens/WriterReaderExample.kt

```diff
@@ -28,7 +33,7 @@ object ConsumptionPlan : Plan(
             "",
             mapOf(
                 "data" to HandleConnection(
-                    StorageKeyParser.parse("reference-mode://{db://25e71af4e9fc8b6958fc46a8f4b7cdf6b5f31516@arcs/Thing}{db://25e71af4e9fc8b6958fc46a8f4b7cdf6b5f31516@arcs/!:writingArcId/handle/my-handle-id}"),
+                    StorageKeyParser.parse("db://25e71af4e9fc8b6958fc46a8f4b7cdf6b5f31516@arcs/!:writingArcId/handle/my-handle-id"),
```

Comment:
why isn't it a reference-mode-storage key anymore?

Reply:
I'm curious about this, too. I'm running the bazel rule to generate this, and I keep getting something different than what I get from calling the js directly.

Reply:
ts*

Reply:
Resolved in previous PR.